src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -2706,17 +2735,17 @@ void cannotCreateRequestWithoutInstanceId(RequestLevel requestLevel) {
       .withPickupServicePointId(servicePointsFixture.cd1().getId()));
 
     assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
-    assertThat(postResponse.getJson(), hasErrors(requestLevel == RequestLevel.ITEM ? 1 : 2));
```

Comment:
Try split this test in two: Item and Title

Reply:
done